temel_bilgiler/giriş/web-geliştirmeye-giriş.md

```diff
@@ -0,0 +1,97 @@
+### Giriş
+
+Web geliştiricinin görevi nedir? Bir bakıma, websitesi yapma ve onun bakımından sorumludurlar.
+
+Web geliştiriciler genelde ürününü veya hizmetini web'e taşımaya çalışan müşteriler için çalışırlar. Çalışma genelde proje odaklıdır ve müşterinin ihtiyaçlarını son ürüne koordine eden bir ekip ile işbirliği yapmayı içerir. Müşteri bir teknoloji şirketi, bir organizasyon veya bir hükümet olabilir. Çalışma Frontend, Backend veya FullStack web geliştirme içerebilir.
+
+Web geliştirme eğer mantıksal problemleri çözmeyi seviyor, yeni kullanışlı şeyler yapmayı seviyor ve yeni teknolojileri denemeyi seviyorsanız sizin için iyi bir meslek olabilir. Web geliştiriciler genelde iyi bir iş/yaşam dengesine sahiptir, diğer iş alanlarına göre daha dolgun maaş alırlar ve yüksek talep görürler. Web geliştirme iş fırsatlarınızı daha iyi anlamak için özel konumunuzu Google'da aratın.
+
+Web geliştirme hakkında daha fazla bilgi için, Wikipedia da [web dizayn a giriş](https://en.wikipedia.org/wiki/Web_design)'e bakabilirsiniz .
+
+### Web Geliştirme Türleri
+
+Önceden, web geliştirme işinin Frontend, Backend veya FullStack olabileceğinden bahsetmiştik. Bu terimler tam olarak ne anlama geliyor?
+
+Frontend, tarayıcınızda gördüğünüz website'deki içerik ve kullanıcı arayüzü elemanları, örneğin navigasyon barı gibi şeylerin sunumunu içerir. Frontend geliştiricileri HTML, CSS, JavaScript ve ilgili framework'leri kullanarak içeriğin etkili bir şekilde sunulmasını ve kullanıcıların mükemmel bir deneyim yaşamasını sağlar.
+
+Backend, uygulamanın iç kısmını ifade eder, bu kısım sunucuda bulunur. Backend, ön uçun ihtiyaç duyduğu şeyleri sağlamak için program verilerini depolar ve sunar. Bu süreç, bir website'nin milyonlarca kullanıcısı olduğunda çok karmaşık hale gelebilir. Backend geliştiricileri veri ile çalışmak için Java, Python, Ruby ve JavaScript gibi programlama dillerini kullanır.
```

Comment:
**Arkayüz**, uygulamanın sunucuda yaşayan iç kısmını ifade eder. Arkayüz, önyüzün ihtiyaç duyduğu şeyleri sağlamak için program verilerini depolar ve sunar. Bir internet sitesinin milyonlarca kullanıcısı olduğunda bu süreç çok karmaşıklaşabilir. Arkayüz geliştiricileri verilerle çalışmak için Java, Python, Ruby ve JavaScript gibi programlama dillerini kullanır.